### ISSUE TITLE
Support for Nextcloud > 18

### ISSUE THREAD

**Issue by sunjam** [nextcloud-gmbh/files_frommail#40]
Please add support for version 19 (and 20 which will be in Stable shortly).  Thanks!

**Comment by nursoda** [comment]
I've set this up in NC 18 and migrated to NC 19 and 20, no issues – so from an "admin that is just using this to solve a specific issue" perspective, there's nothing that needs to be changed except the max version to support NC >18. For my instances it currently is the only app left that needs an "activate untested app" :/

Please also check if you may set max version to NC 21. I had very little issues migrating my test instance to NC 21 Beta 1 :)

**Comment by alejandroscf** [comment]
It would be great to have this App supported in NC 19.

**Comment by nursoda** [comment]
@sunjam please change subject to "Support for Nextcloud >18" since NC21 already has a release candidate.

**Comment by huckley** [comment]
have anybody running this on  NC22?

**Comment by nursoda** [comment]
Yes, works fine for me on NC 22 on arch with all up-to-date versions.
